src/gnome-utils/ui/gnucash-fallback.css

```diff
@@ -0,0 +1,4 @@
+/* Note: Widgets obtained from Glade files will not be addressable
+   directly by name as these are not brought in when loaded, only
+   the widget type can be configured unless they are named in code */
```

Comment:
Looks like the widgets need to be named in the glade files, then you can use some thing like #imap-dialog or you can add classes to them